react/src/utils/hooks.ts

```diff
@@ -11,19 +16,25 @@ export interface INotificationProps extends UseToastOptions {
 
 export const useNotification = () => {
   const toast = useToast();
+  const [isSmallScreen] = useMediaQuery("(max-width: 1070px)");
+  const [position, setPosition] = useState<ToastPositionWithLogical>("bottom");
+
+  useEffect(() => {
+    setPosition(isSmallScreen ? "bottom" : "top");
+  }, [isSmallScreen]);
 
   const createToast = useCallback(
     ({ message, type, ...props }: INotificationProps) =>
       toast({
         duration: 5000,
         isClosable: true,
-        position: "bottom",
+        position: position,
```

Comment:
I rebased on latest master and the project does not build anymore. In this line I get an eslint complaint `Expected property shorthand  object-shorthand`